src/api.ts

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import axios from "axios";
+const striptags: (html: string, allowedTags?: string | string[]) => string = require("striptags");
```

Comment:
Unfortunately, the striptags type def uses `export.default`, which as far as I can tell is incompatible with commonjs-style imports. I added the signature from the typedef file, though, which accomplishes the same goal.